library/aes.c

```diff
@@ -417,7 +427,11 @@ static void aes_gen_tables(void)
      * generate the forward and reverse S-boxes
      */
     FSb[0x00] = 0x63;
+#if (!defined(MBEDTLS_AES_DECRYPT_ALT) || !defined(MBEDTLS_AES_SETKEY_DEC_ALT)) && \
```

Comment:
Would it be better to have a macro internal to this file `MBEDTLS_AES_NEED_REVERSE_S_BOXES`?

(And don't need to comment on `#endif` when it's so close to the `#ifdef`

Reply:
I think with all these changes the `aes.c` files in 2.28 and `development` are going to diverge, so no need to backport - it's not a bug fix, just a refactor for readability and maintainability